src/components/setup_meetings/RoomSetupMeeting.js

```diff
@@ -1,40 +1,59 @@
 "use-strict";
 import React, { Component } from "react";
-import { View, Text, FlatList, Dimensions, StyleSheet, SafeAreaView } from "react-native";
-import HeaderSetupMeeting from "../setup_meetings/HeaderSetupMeeting"
-
+import {
+  View,
+  Text,
+  FlatList,
+  Dimensions,
+  StyleSheet,
+  SafeAreaView
+} from "react-native";
+import HeaderSetupMeeting from "../setup_meetings/HeaderSetupMeeting";
+import RowSetup from "./RowSetup";
 
 class RoomSetupMeeting extends Component {
   render() {
     return (
       <SafeAreaView style={{ flex: 1 }}>
-        <View style={{ flex: 1 }}>
+        <View style={styles.headerContainer}>
           <HeaderSetupMeeting />
         </View>
-        <View style={styles.row}>
-          <Text>Date</Text>
-        </View>
-        <View style={styles.row}>
-          <Text>Time</Text>
-        </View>
-        <View style={styles.row}>
-          <Text>Recurrency</Text>
-        </View>
-        <View style={styles.row}>
-          <Text>Add people</Text>
-        </View>
-        <View style={styles.row}>
-          <Text>add description</Text>
+        <View style={styles.rowsContainer}>
+          <View style={styles.row}>
+            <RowSetup />
+          </View>
+          <View style={styles.row}>
```

Comment:
maybe we could create a component for this, instead of passing the styles every time:

```tsx
<RoomRow>
  <RowSetup />
</RoomRow>
```

Reply:
Done